libs/zipfile.js

```diff
@@ -322,7 +322,7 @@ function inflate(bytes) {
 
 var arrays = J2ME.ArrayUtilities.makeArrays(256);
 
-function ZipFile(buffer) {
+function ZipFile(buffer, extract) {
```

Comment:
The *extract* argument name is a bit misleading. It doesn't actually extract files from the JAR, it just creates different ArrayBuffers for each file (instead of different views on the same ArrayBuffer).